gds/src/main/scala/edu/gemini/aspen/gds/keywords/ConstantKeywordCollector.scala

```diff
@@ -0,0 +1,29 @@
+package edu.gemini.aspen.gds.keywords
+
+import cats.effect.Async
+import cats.syntax.all._
+import edu.gemini.aspen.gds.configuration.{ KeywordConfiguration, KeywordConfigurationItem }
+import edu.gemini.aspen.gds.fits.FitsValue
+import edu.gemini.aspen.gds.fits.FitsType.StringType
+import edu.gemini.aspen.gds.model.{ GdsError, KeywordSource }
+
+object ConstantKeywordCollector {
+  def apply[F[_]](config: KeywordConfiguration)(implicit F: Async[F]): KeywordCollector[F] = {
```

Comment:
Do we need `Async` here?. It seems `ApplicativeError` could be enough?

Reply:
`ApplicativeError` would be enough for the code in this file, but the call to `KeywordCollector.apply` requires an `Async` instance for `F`.